File: Selenium/Activities/TestActivity6.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.color import Color

# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://training-support.net/webelements/dynamic-controls")
    # Print the title of the page
    print("Page title is: ", driver.title)

    checkbox = driver.find_element(By.XPATH,"//input[@id='checkbox']")
    
    print("is checkbox visible ? ",checkbox.is_selected())
    checkbox.click()
    print("is checkbox visible ? ",checkbox.is_selected())

    driver.quit()

File: Selenium/Activities/TestActivity6.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class TestActivity6 {

	public static void main(String[] args) {
WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-controls");
        // Print the title of the page
        System.out.println("Page title: " + driver.getTitle());
        
        WebElement checkbox = driver.findElement(By.xpath("//input[@id='checkbox']"));
        System.out.println("Is the checkbox selected "+ checkbox.isSelected());
       
        checkbox.click();
        System.out.println("Is the checkbox selected "+ checkbox.isSelected());
        
        driver.quit();
        

	}

}
